phoebe/frontend/bundle.py

```diff
@@ -4246,9 +4246,9 @@ def _get_surf_area(comp):
 
             # 2.2 disables support for boosting.  The boosting parameter in 2.2 only has 'none' as an option, but
             # importing a bundle from old releases may still have 'linear' as an option, so we'll check here
-            if compute_kind in ['phoebe'] and self.get_value(qualifier='boosting_method', compute=compute, boosting_method=kwargs.get('boosting_method', None), **_skip_filter_checks) != 'none':
+            if compute_kind in ['phoebe'] and self.get_value(qualifier='boosting_method', compute=compute, boosting_method=kwargs.get('boosting_method', None), **_skip_filter_checks) not in ['none','manual']:
                 report.add_item(self,
-                                "support for beaming/boosting has been removed from PHOEBE 2.2.  Set boosting_method to 'none'.",
+                                "support for interpolated ('linear') beaming/boosting has been removed since PHOEBE 2.2.  Set boosting_method to 'none' or 'manual'.",
```

Comment:
(or something similar... this is hard to word concisely)

```suggestion
                                "support for interpolated ('linear') beaming/boosting in compute options has been removed since PHOEBE 2.2.  'manual' boosting is now supported in the dataset options instead.",
```